### PR TITLE
Allow returning base device model names

### PR DESCRIPTION
**Pull request by armcknight**
and some other cleanup of user-facing outputs for consistency

### CODE REVIEW

File: DeviceList.plist [hunk 1]
```diff
@@ -6,274 +6,352 @@
...
+		<string>iPhone 4</string>
 		<key>name</key>
 		<string>iPhone 4 (CDMA)</string>
 	</dict>
 	<key>iPhone4,1</key>
 	<dict>
 		<key>version</key>
 		<real>4.1</real>
+		<key>baseName</key>
+		<string>iPhone</string>
```
Comment (alexrrouse) [inline DeviceList.plist]: Lets do `iPhone 4S`

File: DeviceList.plist [hunk 2]
```diff
@@ -286,323 +364,415 @@
...
 		<key>name</key>
-		<string>iPad Pro 11-Inch (Wi-Fi/Cellular)</string>
+		<string>iPad Pro 11″ (Wi-Fi + Cellular)</string>
 	</dict>
 	<key>iPad8,4</key>
 	<dict>
 		<key>version</key>
 		<real>8.4</real>
+		<key>baseName</key>
+		<string>iPad Pro 11″ 1TB</string>
```
Comment (alexrrouse) [inline DeviceList.plist]: lets drop the `1TB`

**Review by alexrrouse — APPROVED** [review]
Looks good to me!

File: DeviceList.plist (hunk 1 above)
Comment (armcknight) [reply DeviceList.plist]: Great catch ⚾️

**Pull request merged**